README.md

```diff
@@ -165,7 +165,7 @@ Novu provides a single API to manage providers across multiple channels with a s
 - [x] [Burst SMS](https://github.com/novuhq/novu/tree/main/providers/burst-sms)
 - [x] [Firetext](https://github.com/novuhq/novu/tree/main/providers/firetext)
 - [x] [Infobip](https://github.com/novuhq/novu/tree/main/providers/infobip)
-- [x] [SNS](https://github.com/novuhq/novu/tree/main/providers/sns)
+- [x] [Pinpoint](https://github.com/novuhq/novu/tree/main/providers/pinpoint)
```

Comment:
I think SNS and pinpoint are different providers, so it should be added as new line.

Reply:
https://github.com/novuhq/novu/pull/3119/files/b5d764e50d1f4744ba3764779c2a742f8152de3b#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L155

SNS was listed twice so I overwrote one mention. It's still listed in line 155.

Reply:
Thanks for update!